openstates/co/bills.py

```diff
@@ -299,7 +299,11 @@ def scrape_votes(self, bill, page):
     def scrape_vote(self, bill, vote_url, chamber, date):
         page = self.lxmlize(vote_url)
 
-        motion = page.xpath('//td/b/font[text()="MOTION:"]/../../following-sibling::td/font/text()')[0]
+        try:
+            motion = page.xpath('//td/b/font[text()="MOTION:"]/../../following-sibling::td/font/text()')[0]
+        except:
+            motion = 'withdrawn'
```

Comment:
`Adopt amendment L.002.`  has blank `Vote Summary` page. [Link](http://leg.colorado.gov/bills/hb17-1293). So whenever it will try to access the `motion` on a blank page I set the `motion= 'withdrawn'` so that it will not scrape the page further (`if 'withdrawn' not in motion:`) to avoid error. manually setting the `motion` will not affect the real database.